daemon/response.go

```diff
@@ -74,9 +75,26 @@ func (r *resp) transmitMaintenance(kind errorKind, message string) {
 //      The right code style takes a bit more work and unifies
 //      these fields inside resp.
 type Meta struct {
```

Comment:
Offtopic (maybe) - what is the Meta structure for? The name is a bit ... meta

Reply:
information about the request, or the response, or snapd in general, but not the response itself